spaceship/lib/spaceship/du/du_client.rb

```diff
@@ -15,49 +15,59 @@ def self.hostname
     #####################################################
 
     def upload_screenshot(app_version, upload_file, content_provider_id, sso_token_for_image, device, is_messages)
-      upload_file(app_version, upload_file, '/upload/image', content_provider_id, sso_token_for_image, screenshot_picture_type(device, is_messages))
+      upload_file(app_version, upload_file, '/upload/image', content_provider_id, sso_token_for_image, screenshot_picture_type(device, is_messages), false)
+    end
+
+    def upload_purchase_review_screenshot(app_id, upload_file, content_provider_id, sso_token_for_image)
+      upload_file(app_id, upload_file, '/upload/image', content_provider_id, sso_token_for_image, 'MZPFT.SortedScreenShot', true)
     end
 
     def upload_large_icon(app_version, upload_file, content_provider_id, sso_token_for_image)
-      upload_file(app_version, upload_file, '/upload/image', content_provider_id, sso_token_for_image, 'MZPFT.LargeApplicationIcon')
+      upload_file(app_version, upload_file, '/upload/image', content_provider_id, sso_token_for_image, 'MZPFT.LargeApplicationIcon', false)
     end
 
     def upload_watch_icon(app_version, upload_file, content_provider_id, sso_token_for_image)
-      upload_file(app_version, upload_file, '/upload/image', content_provider_id, sso_token_for_image, 'MZPFT.GizmoAppIcon')
+      upload_file(app_version, upload_file, '/upload/image', content_provider_id, sso_token_for_image, 'MZPFT.GizmoAppIcon', false)
     end
 
     def upload_geojson(app_version, upload_file, content_provider_id, sso_token_for_image)
-      upload_file(app_version, upload_file, '/upload/geo-json', content_provider_id, sso_token_for_image)
+      upload_file(app_version, upload_file, '/upload/geo-json', content_provider_id, sso_token_for_image, false)
     end
 
     def upload_trailer(app_version, upload_file, content_provider_id, sso_token_for_video)
-      upload_file(app_version, upload_file, '/upload/purple-video', content_provider_id, sso_token_for_video)
+      upload_file(app_version, upload_file, '/upload/purple-video', content_provider_id, sso_token_for_video, false)
     end
 
     def upload_trailer_preview(app_version, upload_file, content_provider_id, sso_token_for_image)
-      upload_file(app_version, upload_file, '/upload/app-screenshot-image', content_provider_id, sso_token_for_image)
+      upload_file(app_version, upload_file, '/upload/app-screenshot-image', content_provider_id, sso_token_for_image, false)
     end
 
     private
 
-    def upload_file(app_version, upload_file, path, content_provider_id, sso_token, du_validation_rule_set = nil)
+    def upload_file(app_version, upload_file, path, content_provider_id, sso_token, du_validation_rule_set = nil, use_app_id = false)
```

Comment:
Not very fond of the app_version used to contain 2 different types.

I think I would prefer an explicit `app_id` parameter
```
def upload_file(app_version, upload_file, path, content_provider_id, sso_token, du_validation_rule_set = nil, app_id = nil)
```
or maybe better 2 methods
```
# renamed from upload_file()
def upload_app_version_file(app_version: ...)
  upload_app_file(app_id: app_version.version, ....)
end
def upload_app_file(app_id, version: nil, app_type: ..., .... ) # note the : instead of =
```